wiki/Tournaments/OWC/2013/en.md

```diff
@@ -3,426 +3,393 @@ tags:
+  - [paraoka - Manima ni (Short Ver.) (Mixagji) \[0108\]](https://osu.ppy.sh/beatmaps/131362)
+  - [syatten remixed celas - Bird Sprite -Awakening of Light- (DaxMasterix) \[Another\]](https://osu.ppy.sh/beatmaps/135177)
 - HardRock
-  - [momori - Togameru Kage (cRyo\[iceeicee\]) \[Insane\]](https://osu.ppy.sh/b/231988)
-  - [Otokaze - Karen (Short Ver.) (spboxer3) \[Hanabi\]](https://osu.ppy.sh/b/154357)
-  - [Hanatan - Kotoba Tsunagi (terametis) \[Insane\]](https://osu.ppy.sh/b/157735)
+  - [momori - Togameru Kage (cRyo\[iceeicee\]) \[Insane\]](https://osu.ppy.sh/beatmaps/231988)
+  - [Otokaze - Karen (Short Ver.) (spboxer3) \[Hanabi\]](https://osu.ppy.sh/beatmaps/154357)
+  - [Hanatan - Kotoba Tsunagi (terametis) \[Insane\]](https://osu.ppy.sh/beatmaps/157735)
 - DoubleTime
-  - [SYNC.ART'S feat. Sakaue Nachi - Taketori Hishou (S i R i R u) \[Lunatic\]](https://osu.ppy.sh/b/73384)
-  - [Sasaki Sayaka - Zzz (Sumisola) \[Insane\]](https://osu.ppy.sh/b/105950)
-  - [Golden City Factory - Twilight Chronicle ~ I am Sister (Patchouli) \[Lunatic\]](https://osu.ppy.sh/b/86142)
+  - [SYNC.ART'S feat. Sakaue Nachi - Taketori Hishou (S i R i R u) \[Lunatic\]](https://osu.ppy.sh/beatmaps/73384)
+  - [Sasaki Sayaka - Zzz (Sumisola) \[Insane\]](https://osu.ppy.sh/beatmaps/105950)
+  - [Golden City Factory - Twilight Chronicle ~ I am Sister (Patchouli) \[Lunatic\]](https://osu.ppy.sh/beatmaps/86142)
 - FreeMod
-  - [Blue Stahli - Shotgun Senorita (Zardonic Remix) (Aleks719) \[Insane\]](https://osu.ppy.sh/b/192508)
-  - [Hatsune Miku - Marionette no Kairaku (rui) \[Uncontrollable\]](https://osu.ppy.sh/b/139652)
-  - [Beridzebeth - Seijin no Tou (Strawberry) \[Another\]](https://osu.ppy.sh/b/194953)
+  - [Blue Stahli - Shotgun Senorita (Zardonic Remix) (Aleks719) \[Insane\]](https://osu.ppy.sh/beatmaps/192508)
+  - [Hatsune Miku - Marionette no Kairaku (rui) \[Uncontrollable\]](https://osu.ppy.sh/beatmaps/139652)
+  - [Beridzebeth - Seijin no Tou (Strawberry) \[Another\]](https://osu.ppy.sh/beatmaps/194953)
 - Tiebreaker
-  - **[DJ Okawari - Luv Letter (nold\_1702) \[Posthumous\]](https://osu.ppy.sh/b/127363)**
+  - **[DJ Okawari - Luv Letter (nold\_1702) \[Posthumous\]](https://osu.ppy.sh/beatmaps/127363)**
 
-------------------------------------------------------------------------
-
-Match Results
---------------
+## Results
 
 ### Finals
 
-**07\. Dec 2013 (Saturday)**
-
-| Team A                                   | Score      | Team B                                | History                             |
-|:-----------------------------------------|:----------:|--------------------------------------:|-------------------------------------|
-| ![][flag_KR] **Korea** | **6**  - 5 | Taiwan ![][flag_TW] | [#1](https://osu.ppy.sh/mp/3233030) |
+| Saturday, 2013-12-07 | | | | |
+| ---: | :---: | :---: | :--- | :---: |
+| ![][flag_KR] **Korea** | **6** | 5 | Taiwan ![][flag_TW] | [#1](https://osu.ppy.sh/community/matches/3233030) |
```

Comment:
A pain to do but we are putting the flags at the middle, so this (and every match) should be:
`| **Korea** ![][flag_KR] | **6** | 5 | ![][flag_TW] Taiwan |`